vms/registration/views.py

```diff
@@ -16,7 +16,7 @@
 from administrator.models import *
 
 
-class AdministratorSignupView(TemplateView):
+class AdministratorSignupView(AnonymousRequiredMixin, TemplateView):
```

Comment:
@necessary129 No, that view should be accessible to the admin and not the volunteer. A logged in admin should be able to access. Please remove the mixin .